### ISSUE TITLE
Why are the generated tests run under taint mode?

### ISSUE THREAD

**Issue by hakonhagland** [xsawyerx/module-starter#74]
I noticed that the generated tests in the `t/` directory has shebang `#!perl -T` which causes the test `t/00-load.t` to fail when my module uses `Alien::GSL`. I get the error:
```
$ perl -T -I./lib t/00-load.t 
not ok 1 - use My::Module;
#   Failed test 'use My::Module;'
#   at t/00-load.t line 10.
#     Tried to use 'My::Module'.
#     Error:  Insecure dependency in chdir while running with -T switch at /home/hakond/perl5/perlbrew/perls/perl-5.32.0/lib/5.32.0/File/Find.pm line 375.
# BEGIN failed--compilation aborted at lib/My/Module.pm line 7.
# Compilation failed in require at t/00-load.t line 10.
# BEGIN failed--compilation aborted at t/00-load.t line 10.
```
I am not so familiar with taint mode and wonder if it is the tests that should be changed (remove the `-T` option) or the module `Alien::GSL`?

**Comment by Grinnz** [comment]
Sorry, this was fixed in https://github.com/xsawyerx/module-starter/commit/0b8e7cb51563085621df7ec4e0aa9b7de1be186c but is not yet in a release. I will get one out soon. Thanks

**Comment by Grinnz** [comment]
This is now released in 1.77.
